### ISSUE TITLE
Direct link to the locally archive webpage (from the listing page)

### ISSUE THREAD

**Issue by JamesMenetrey** [sissbruecker/linkding#867]
Dear @sissbruecker,

Many thanks for your great app, especially the recent feature #672 !

As a huge fan of this new feature, it would be handy (at least for me) to have the ability to change the archive link (from the listing page) to use the locally archived version instead of the Web archive website (see the attached screenshot for the link I mention).

Indeed, having a direct link to the local archive version would speed up the usage of my local archive instead of clicking "view" and selecting the latest local archive. You can propose a toggle on the settings page to see if a web archive or local archive must be used for this link.

If you plan to address this suggestion, I highly suggest you to link the latest HTML archive of the local archive. This means that if one manually uploads an HTML archive (typically from an authenticated webpage), this link would point to that manually uploaded document.

Cheers!

<img width="758" alt="link_for_archive" src="https://github.com/user-attachments/assets/07d6a18f-e53d-45e4-abdc-2957a9e704dd">

**Comment by yellowbikewithbluehandles** [comment]
Yes I would love this as well because if I disable url validation and put it in a local file directly even if you click it it will not open the html to the local archive.
